src/main/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtils.java

```diff
@@ -546,4 +494,24 @@ public static String getDictionaryAsString( final SAMSequenceDictionary dict ) {
 
         return s.toString();
     }
+
+    /**
+     * get the sequence dictionary contig list that is in the index or null if there is not index or not contigs
```

Comment:
Document that this method is specifically for getting sequence dictionaries from indices on Feature files (tribble or tabix indices).